actions/get-reddit-posts/output-schema.js

```diff
@@ -1,4 +1,6 @@
-module.exports = [ // the action function must return an object with the following parameters
+// These are used to help Clay's interface provide ways to extract data from the output of this action into the table
```

Comment:
Should we just generate this in the deploy endpoint automatically unless it's provided.